speed-dreams/source-2.2.3/data/data/menu/ResearcherMenu.xml

```diff
@@ -6,31 +6,27 @@
 
     <section name="dynamic controls">
 
-        <!-- Checkboxes for selecting a Task -->
-        <section name="CheckboxTaskLaneKeeping">
-            <attstr name="type" val="check box"/>
-            <attstr name="text" val="Lane keeping"/>
-            <attstr name="tip" val="Select the Lane Keeping task"/>
+        <!-- RadioButtonList for selecting a Task -->
+        <section name="TaskRadioButtonList">
+            <attstr name="type" val="radio button list"/>
+            <section name="sub texts">
+                <attstr name="text0" val="Lane Keeping"/>
+                <attstr name="text1" val="Speed Control"/>
+            </section>
+            <section name="sub tips">
+                <attstr name="tip0" val="Select the Lane Keeping task"/>
+                <attstr name="tip1" val="Select the Speed Control task"/>
+            </section>
             <attnum name="x" val="20"/>
             <attnum name="y" val="370"/>
             <attnum name="image width" val="20"/>
             <attnum name="image height" val="20"/>
             <attstr name="color" val="0xFFFFFF"/>
             <attstr name="font" val="small_t"/>
-            <attstr name="checked" val="no"/>
-        </section>
-
-        <section name="CheckboxTaskSpeedControl">
-            <attstr name="type" val="check box"/>
-            <attstr name="text" val="Speed control"/>
-            <attstr name="tip" val="Select the Speed Control task"/>
-            <attnum name="x" val="20"/>
-            <attnum name="y" val="340"/>
-            <attnum name="image width" val="20"/>
-            <attnum name="image height" val="20"/>
-            <attstr name="color" val="0xFFFFFF"/>
-            <attstr name="font" val="small_t"/>
-            <attstr name="checked" val="no"/>
+            <attnum name="selected" val="0"/>
+            <attnum name="amount" val="2"/>
```

Comment:
Remove. Get the amount in code by calling `GfParmGetEltNb(handle, path_to_options_section)`
```suggestion
```